### PR TITLE
feat(provider/google): Support TCP Proxy Load Balancing

### PR DESCRIPTION
**Pull request by sayap**
Mostly copy-paste from SSL Proxy code.

### CODE REVIEW

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/view/GoogleLoadBalancerProvider.groovy [hunk 1]
```diff
@@ -259,7 +270,11 @@ class GoogleLoadBalancerProvider implements LoadBalancerProvider<GoogleLoadBalan
         loadBalancerPort = portString
         break
       case GoogleLoadBalancerType.SSL:
-        instancePort = 'http' // NOTE: This is what occurs in Google Cloud Console, it's not documented and a bit non-sensical.
```
Comment (sayap) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/view/GoogleLoadBalancerProvider.groovy]: I believe this is just for display purpose in deck. For SSL/TCP proxies, both instance port and load balancer port must be the same.

**Comment by sayap** [comment]
PR for deck: https://github.com/spinnaker/deck/pull/3894

**Comment by sayap** [comment]
Thanks for the review. I have added separate commits to address the issues.

**Comment by sayap** [comment]
Rebased. Need to squash as well?

**Comment by jtk54** [comment]
No need, I can squash & merge.

**Comment by jtk54** [comment]
Thanks for the contribution, I'll ping the UI PR.

**Pull request merged**